README.md

```diff
@@ -58,26 +52,25 @@ Argument Reference
 
 The following arguments are supported:
 
-* `domain` - (Required) Domain name of the certificate
+* `domain` - (Required) Domain name of the certificate.
 
-* `hostname` - (Required) Hostname of the certificate
+* `hostname` - (Required) Hostname of the certificate.
 
 * `validation_method` - Which method to use for validation. DNS or EMAIL are valid, NONE can be used for certificates that were imported into ACM and then into Terraform.
 
 * `subject_alternative_names` - A list of domains that should be SANs in the issued certificate
 
-* `skip_route53_validation` - Set to true for zones not hosted in Route53
+* `skip_route53_validation` - Skip Route53 validation. Default is false, which creates Route53 domain validation records to be created. Set to true for zones not hosted in Route53.
 
-* `create_route53_record` - Set to false if Route53 record already exists
+* `create_route53_record` - Create Route53 record. Default is true. Set to false if Route53 record already exists.
```

Comment:
I recommend for consistency:

* `create_route53_record` - Create Route53 record if true. Set to false if Route53 record already exists (Default is true).